src/ddqa/cache/github.py

```diff
@@ -98,3 +98,12 @@ def cache_candidate_data(self, commit_hash: str, candidate_data: dict):
             (directory / candidate_data['id']).touch()
         else:
             (directory / 'no_pr.json').write_text(json.dumps(candidate_data, cls=SetEncoder))
+
+    def get_team_members(self, team: str) -> set[str] | None:
+        if (members_file := self.get_team_members_file(team)).is_file():
```

Comment:
I think this would be much clearer without inlining the assignment with `:=`, as it makes it a bit too dense.

Reply:
You're right!